setup.cfg

```diff
@@ -0,0 +1,16 @@
+[metadata]
+description-file = README.rst
+
+[bdist_wheel]
+universal=1
+
+[flake8]
+# https://flake8.readthedocs.org/en/latest/
+exclude=docs/*,build/*
+max-complexity = 15
+max-line-length = 119
+
+[pep8]
+# https://pep8.readthedocs.org/en/latest/
+# W503 raised if operators are after line break
+ignore = W503
```

Comment:
If we're going to use flake8, there are a few more warnings that need to be disabled, such as the ones about star-imports. I have a flake8 config lying around in a git stash somewhere, I should make a PR for that later...

Reply:
I haven't tied flake8 into the build process (yet); so this is just groundwork. If you've got any suggestions for things to turn on/off, PRs welcome.